ci-operator/config/terraform-redhat/terraform-provider-rhcs/terraform-redhat-terraform-provider-rhcs-main.yaml

```diff
@@ -148,6 +104,30 @@ tests:
     test:
     - ref: rhcs-e2e-tests
     workflow: rhcs-aws-sts
+- as: rosa-sts-ad-critical-high-presubmit
```

Comment:
Please add a description 
Also, what is it `ad`?

Reply:
unfortunately, all the comments deleted by the script(